packages/cli/src/commands/install/install.js

```diff
@@ -9,14 +9,14 @@
 
 import type {ContextT} from '../../tools/types.flow';
 import logger from '../../tools/logger';
-import * as PackageManager from '../../tools/PackageManager';
+import * as PackageManager from '../../tools/packageManager';
```

Comment:
you changed module name from `PackageManager` to `packageManager`, but the variable name stayed without changes. mind fixing?

Reply:
Variable name left uppercased because it is a grouping of all methods in `packageManager` through `* as` importing.